platforms/nuttx/CMakeLists.txt

```diff
@@ -137,11 +137,19 @@ CYGPATH(NUTTX_CONFIG_DIR NUTTX_CONFIG_DIR_CYG)
 if((DEFINED ENV{SIGNING_TOOL}) AND (NOT NUTTX_DIR MATCHES "external"))
 	set(PX4_BINARY_OUTPUT ${PX4_BINARY_DIR}/${PX4_CONFIG}_unsigned.bin)
 
-	add_custom_command(OUTPUT ${PX4_BINARY_DIR_REL}/${PX4_CONFIG}.bin
-		COMMAND $ENV{SIGNING_TOOL} $ENV{SIGNING_ARGS} ${PX4_BINARY_OUTPUT} ${PX4_BINARY_DIR}/${PX4_CONFIG}.bin
-		DEPENDS ${PX4_BINARY_OUTPUT}
-		WORKING_DIRECTORY ${PX4_SOURCE_DIR}
-	)
+	if (CONFIG_OPENSBI AND "${PX4_BOARD_LABEL}" STREQUAL "bootloader")
```

Comment:
The bootloader is signed, but I don't think it uses the ToC?

Reply:
Actually the bootloader shouldn't be signed with this script; bootloader signature is chip/vendor specific. For us it is signed with microsemi tools at the time of flashing/payload generation

Reply:
Ok, previously the bootloader _was_ signed. This rule does not test for bootloader target -> every target is signed if ENV{SIGNING_TOOL} is defined. I'll add test for bootloader so the signature is not added for it.